### PR TITLE
Update the messaging for the international edition.

### PR DESCRIPTION
**Pull request by gklopper**

### CODE REVIEW

File: common/app/common/LinkTo.scala [hunk 1]
```diff
@@ -141,3 +142,14 @@ object AnalyticsHost extends implicits.Requests {
...
 
+object SubscribeLink {
+  private val subscribeEditions = Map(
+    Us -> "us",
+    Au -> "au"
+  )
+
+  private def subscribeLink(edition: Edition) = subscribeEditions.getOrDefault(edition, "")
+
+  def apply(edition: Edition): String = s"https://subscribe.theguardian.com/${subscribeLink(edition)}?INTCMP=NGW_HEADER_${edition.id}_GU_SUBSCRIBE"
```
Comment (gklopper) [inline common/app/common/LinkTo.scala]: cc @dominickendrick this also fixes the international edition subscribe link.

File: static/src/javascripts/projects/common/views/international-message.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <p class="site-message__message" id="site-message__message">
-    You’re viewing a beta release of the Guardian’s international edition.
+    You are viewing the Guardian’s international edition.
 </p>
 <ul class="site-message__actions u-unstyled">
     <li class="site-message__actions__item">
         <i class="i i-back"></i>
-        <a class="js-main-site-link" rel="nofollow" href="/preference/edition/uk" data-link-name="opt-out">Opt out and go to UK edition</a>
+        <a class="js-main-site-link" rel="nofollow" href="/preference/edition/uk" data-link-name="opt-out">Opt out and visit the UK edition instead</a>
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/views/international-message.html]: it is an opt out, or just a difference?  I'd consider saying simply "Visit the UK edition instead"

**Comment by johnduffell** [comment]
:+1: 1:

**Comment by gklopper** [comment]
Don't mess with the officially approved wording :trollface:

**Pull request merged**